tools/add_triage_signature.py

```diff
@@ -91,12 +91,14 @@ def get_events_json(cluster_url, cluster_id):
     return res.json()
 
 
-def get_cluster_logs(cluster_url: str, cluster_id: str, output_file: str):
-    res = requests.get(f"{cluster_url}/cluster_{cluster_id}_logs.tar")
-    res.raise_for_status()
+def get_host_log_file(log_cache: LRU, cluster_url: str, cluster_id: str, host_id: str, filename: str):
+    tar_url = f"{cluster_url}/cluster_{cluster_id}_logs.tar"
+    remote_nested_tar: nestedarchive.RemoteNestedArchive = log_cache[tar_url]
+
+    # The file is already uniquely determined by the host_id, we can omit the hostname
+    hostname = "*"
```

Comment:
just inline this variable, I don't think it's that important

Reply:
Wanted to add a comment explaining why I do that, if I inline it - there won't be a good place to put that comment. Also just putting `*` doesn't make it really clear that it's the hostname unless you're really familiar with and remember the structure of the logs